curated_transformers/tests/models/test_hf_hub.py

```diff
@@ -13,3 +14,29 @@ def test_sharded_model_checkpoints(torch_device):
     assert_encoder_output_equals_hf(
         BERTEncoder, "explosion-testing/bert-test-sharded", torch_device
     )
+
+
+def test_download_to_cache():
+    BERTEncoder.download_to_cache(
```

Comment:
If the test above runs earlier, it could be that the checkpoints are already downloaded that test. Probably the easiest solution would be to have a separate repo for this that it not used by any other test?

Reply:
Good point. I'll create a separate repo.